cmd/deploy/proxy.go

```diff
@@ -363,6 +363,9 @@ func (ph *proxyHandler) translateStatefulSetSpec(body map[string]json.RawMessage
 		return nil
 	}
 	labels.SetInMetadata(&spec.Template.ObjectMeta, model.DeployedByLabel, ph.Name)
+	for _, pvc := range spec.VolumeClaimTemplates {
+		labels.SetInMetadata(&pvc.ObjectMeta, model.DeployedByLabel, ph.Name)
```

Comment:
Let's do this just on create. Otherwise, an update of an existing sfs will fail because sfs don't allow label changes

Reply:
I think we can do it always. If the stateful set is already deployed on another config map, the user will receive a warning message saying that we will skip that service because it's already deployed, and if the user redeploys the application, the deployed-by label will be the same so no changes will be applied

Reply:
@jLopezbarb but what would happen for a sfs that, being already deployed for a dev environment with a previous version of the CLI, is redeployed using this new code? It would add the new label. I mean, the redeploy would happen for the same configmap so the warning wouldn't be shown.

If I try to include the label in the VolumeClaimTemplate for an already deployed statefulset, I get this error:

```
# statefulsets.apps "postgresql" was not valid:
# * spec: Forbidden: updates to statefulset spec for fields other than 'replicas', 'template', 'updateStrategy' and 'minReadySeconds' are forbidden
```

We should only include the label in the creation